ols/utils/config.py

```diff
@@ -23,130 +23,36 @@ def load_empty_config() -> None:
     llm_config = config_model.LLMConfig()
 
 
-def load_config_from_env() -> None:
-    """Load configuration from environment variables."""
+def load_config_from_stream(stream):
+    """Load configuration from a stream."""
     global ols_config
```

Comment:
IMHO it might be better to try not to use global variables if possible. But it might need some refactoring there

Reply:
It seems that this test does not require the variable ols_config, llm_config, conversation_cache and default_logger to be global to make it working.

Reply:
i agree w/ the sentiment but in this case it's effectively read-only(config is loaded at startup and then nothing should be changing it) so i don't think it's a big problem or that passing it around buys us much.

but it's definitely not a pattern we should be repeating in general.